gcore-spoofax/syntax/Expressions.sdf3

```diff
@@ -67,6 +70,7 @@ context-free syntax // Aggregates
   Exp.max          = <MAX(<Distinct?><Exp>)> {case-insensitive}
   Exp.sum          = <SUM(<Distinct?><Exp>)> {case-insensitive}
   Exp.avg          = <AVG(<Distinct?><Exp>)> {case-insensitive}
+  Exp.group-concat = <GROUP_CONCAT(<Distinct?><Exp>)> {case-insensitive} // concatenation of group of strings using comma (,) as separator
```

Comment:
needed for LDBC SNB Interactive Q1

standard SQL doesn't seem to have a GROUP_CONCAT. Feel free to rename to something else.